.pre-commit-config.yaml

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-exclude: ^(src/grpc_generated|.*\.patch$|.*/connect/.*\.py)
+exclude: ^(src/grpc_generated|.*\.patch$|.*/(nixl_)?connect/.*\.py)
```

Comment:
we shouldn't be exluding these ?

Reply:
this only disables the linting, not the static analysis. basically, this avoids the style-cop like tooling.

I'll take a look and see how much impact it has, and let you know the scope of the results.

Reply:
like to see this in -

Reply:
done. passing all the pre-merge lint stuff. I hate this change. it added over 300 lines to the code and reduced readability significantly imo.

Reply:
thanks!